lexer_test.go

```diff
@@ -1,41 +1,39 @@
-package vdf_test
+package vdf
```

Comment:
Any particular reason, why the package is renamed?
We use a different package here, to be able to test only functions that are exposed and NOT to test inner behavior.

Reply:
Im following the Go testing conventions 

> To write a new test suite, create a file whose name ends _test.go that contains the TestXxx functions as described here. Put the file in the same package as the one being tested. The file will be excluded from regular package builds but will be included when the “go test” command is run. For more detail, run “go help test” and “go help testflag”.

https://golang.org/pkg/testing/